### DISCUSSION TITLE
[New Models] Controlllite for Normal_Dsine [SDXL]

### DISCUSSION THREAD

**Discussion by sdbds** [Mikubill/sd-webui-controlnet#2813] [General]
Match both anime and photorealistic sytle.

#Useage
Just use it as normal_dsine
![image](https://github.com/Mikubill/sd-webui-controlnet/assets/8085926/10eb0c7b-712b-4160-8a86-1984a55e8888)
![image](https://github.com/Mikubill/sd-webui-controlnet/assets/8085926/ad7e8da7-a2fe-4817-b11d-648a0787d9b9)

![image](https://github.com/Mikubill/sd-webui-controlnet/assets/8085926/731a5909-3930-4a38-92f9-058b0964c8e2)
![image](https://github.com/Mikubill/sd-webui-controlnet/assets/8085926/d93b5308-5f11-4d36-a336-a8f439b9e06c)
![image](https://github.com/Mikubill/sd-webui-controlnet/assets/8085926/3f5415db-38c4-4568-b5ea-2d26aa606a48)
![image](https://github.com/Mikubill/sd-webui-controlnet/assets/8085926/4f589d4c-f6c8-4480-a12d-8ed9847418b3)

# Model Download
https://huggingface.co/bdsqlsz/qinglong_controlnet-lllite

**Comment by huchenlei** [comment]
@sdbds You can add the discussion link to https://github.com/Mikubill/sd-webui-controlnet#news section
